test/dot_imports_test.go

```diff
@@ -0,0 +1,18 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/mgechev/revive/lint"
+	"github.com/mgechev/revive/rule"
+)
+
+func TestDotImports(t *testing.T) {
```

Comment:
Can we have a test case with a missing config? To make sure the change is backwards compatible.